crates/object_detection/src/pose_interpretation.rs

```diff
@@ -211,37 +229,53 @@ fn is_shoulder_angled_up(
 
     let shoulder_to_elbow = elbow.coords() - shoulder.coords();
 
-    f32::atan2(shoulder_to_elbow.y(), shoulder_to_elbow.x()) > shoulder_angle_threshold
```

Comment:
I don't understand how this function works at all...
atan2 returns the signed angle to the X axis, so a vector pointing to the right (such as left_shoulder -> left_elbow) should have an angle within `[-pi/2, pi/2]`. A vector pointing to the left (such as right_shoulder -> right_elbow) would have an angle in the range `[-pi, -pi/2] u [pi/2, pi]`.

Furthermore, I don't think the `shoulder_rotation` does what you think it does. You construct a rotation from the angle between the two shoulder points which you then use to rotate the shoulder and elbow points.
However, instead of cancelling the rotation of the whole body posture, it compounds the rotation.
Also, you rotate the points around the origin in pixel coordinates, which works because you only take the difference anyways but makes it confusing to think about.

Instead of all the manual angle calculating, I would suggest something like
```
    Rotation2::rotation_between(left_to_right_shoulder, elbow_point - shoulder_point).angle()
        > minimum_shoulder_angle
```
but this still suffers from the pointing left vs. pointing right issue.